openslides_backend/action/action.py

```diff
@@ -181,16 +185,39 @@ def handle_relation_updates(
         )
         fields: Optional[Dict[str, Any]]
         for fqfield, data in relation_updates.items():
+            field = getattr(model_registry[fqfield.collection], fqfield.field, None)
             list_fields: Optional[ListFields] = None
             if data["type"] in ("add", "remove"):
                 data = cast(FieldUpdateElement, data)
+                if (
+                    field
+                    and field.required
+                    and not data.get("value")
+                    and not isinstance(
+                        self.additional_relation_models.get(fqfield.fqid), DeletedModel
+                    )
+                ):
+                    raise RequiredFieldsException(
+                        f"From relation {fqfield.fqid}", [fqfield.field]
+                    )
```

Comment:
I think by performing the check here you are missing cases where an action directly deletes a field which is required. Or is this caught somewhere else?

Reply:
outdated, this is now detected by checking the WrieRequests